packages/datadog-plugin-aws-sdk/src/services/s3.js

```diff
@@ -18,6 +25,33 @@ class S3 extends BaseAwsSdkPlugin {
       bucketname: params.Bucket
     })
   }
+
+  addSpanPointer (span, response) {
+    const request = response?.request
+    const operationName = request?.operation
+    if (!['putObject', 'copyObject', 'completeMultipartUpload'].includes(operationName)) {
```

Comment:
I know that, at least in python, it's better to do a `if this == that or these == those` rather than asking if something is in an array. This is because the latter does an allocation which takes much more time. Not sure how js does it though!

Reply:
Very interesting indeed! Thanks for sharing this!